recorder/src/main/cpp/config.cc

```diff
@@ -96,17 +96,47 @@ void ConfigurationOptions::load(const char* options) {
                 if (metrics_dst_port == 0) metrics_dst_port = DEFAULT_METRICS_DEST_PORT;
             } else if (strstr(key, "noctx_cov_pct") == key) {
                 noctx_cov_pct = static_cast<std::uint8_t>(atoi(value));
-                if (noctx_cov_pct > 100) noctx_cov_pct = 100;
+                if (noctx_cov_pct > 100) {
+                    logger->warn("NoCtx coverage pct is too high at {}, re-setting it to 100", noctx_cov_pct);
+                    noctx_cov_pct = 100;
+                }
+            } else if (strstr(key, "allow_sigprof") == key) {
+                allow_sigprof = (std::string(value) == "y");
```

Comment:
Understood f2f that we were talking about "y" (the value) and not "allow_sigprof". That seems reasonable. Will do it.